app/controllers/candidate_interface/sign_in_controller.rb

```diff
@@ -56,6 +56,8 @@ def authenticate
           flash[:warning] = "You cannot have more than 3 course choices. You must delete a choice if you want to apply to #{course.name_and_code}."
           redirect_to candidate_interface_course_choices_review_path
         end
+      elsif !FindCandidateByToken.token_not_expired?(candidate)
```

Comment:
Not sure how I feel about this double negative but its really a minor thing 😃

Reply:
Yeah, I don't like the `.token_not_expired?` and not of it either.

Refactor to `.token_expired?` or `.expired_token?`?

Reply:
Yes I think `.token_expired?` sounds better

Reply:
After https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1233#discussion_r373064531 and realising that having to do `!FindCandidateByToken.token_not_expired?(candidate)` is unnecessary, I'm going to leave this refactoring out.